docs/api/sdks/cli.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Using the mittwald CLI
+sidebar_label: CLI
+description: An introduction on how to use the mittwald command-line interface (CLI)
+---
+
+:::danger
+
+Please note that the CLI is currently in an experimental state, and may change without prior notice. We're happy to receive feedback and feature requests via [GitHub issues](https://github.com/mittwald/cli/issues).
+
+:::
+
+## Installing
+
+### Any OS, using NPM
```

Comment:
For macOS und Windows this info is given
> Retrieve the latest release package (`*.exe` file) from our [releases page](https://github.com/mittwald/cli/releases), and run the installer.

But there is no such info for Linux? Is this intentional?

Reply:
Because currently, there is no Linux installer. OCLIF would (theoretically) support building `.deb` and `.rpm` packages, but even those would not be fully equivalent to an "installer". In any case, we'd need to do a bit more testing and tweaking of those packages, before releasing them into the wild.